Source/WebCore/platform/graphics/gbm/DMABufReleaseFlag.h

```
@@ -72,7 +72,9 @@ struct DMABufReleaseFlag {
             return;
 
         uint64_t value { 1 };
+IGNORE_WARNINGS_BEGIN("unused-result")
         write(fd.value(), &value, sizeof(uint64_t));
+IGNORE_WARNINGS_END
```

I would check with one of the graphics developers before assuming that ignoring errors is desired here. Same for the released() function above, actually.

Normally it is better to check for an error as usual and WTFLogAlways if the write failed; ignoring errors is almost always a bug. Notably, neither the read() nor the write() is checking for EINTR, which is _usually_ a bug. My usual strategy is (a) handle EINTR first (with a loop), then (b) for any other error, print a warning with WTFLogAlways() to give anyone debugging problems a chance. If different behavior is desired here, there should probably be a comment to explain why.

---

ACK. I will print a warning to stdout if the write fails.